client/src/App.js

```diff
@@ -1,23 +1,21 @@
-
-import React, { Component } from 'react';
-import SearchBar from './components/search_info_components/search';
-import InfoBar from './components/search_info_components/infoBar';
-import HeaderBar from './components/Header_Bar/header';
-
+import React, { Component } from "react";
+import HeaderBar from "./components/Header_Bar/header";
+import BoxViewContainer from "./components/boxViewContainer";
+import LoginViewContainer from "./components/loginViewContainer";
 class App extends Component {
-render(){
-
-
-  return (
-    <React.Fragment>
-    <HeaderBar />
-    <SearchBar/>
-    <InfoBar />
-    </React.Fragment>
-
-  );
-}
+  state = {
+    login: true
+  };
 
+  render() {
+    return (
+      <React.Fragment>
+        <HeaderBar login={this.state.login} />
+        {this.state.login === true && <BoxViewContainer />}
```

Comment:
Please change into a trinary operator (this line and the next one)

Reply:
done, thanks you.
Next time it will be small PR's :+1: